### PR TITLE
More interesting subject line for instant emails

### PR DESCRIPTION
**Pull request by laurieboyes**
E.g.
screen shot 2016-08-25 at 11 21 48

### CODE REVIEW

File: functions/create-email/test/index.spec.js [hunk 1]
```diff
@@ -74,6 +76,12 @@ describe('Create email', () => {
 			});
 		});
 
+		it('gets the email subject', () => {
+			return invokeFunc.then(() => {
+				expect(mocks.getInstantSubject.calledOnce).to.be.true;
```
Comment (debugwand) [inline functions/create-email/test/index.spec.js]: test returns expected string?

File: README.md [hunk 2]
```diff
@@ -160,6 +160,14 @@ Requires SENT_EMAILS_REDIS_URL env variable
 
 Reads from Kinesis
 
+####Email subject
+
+By default the email subject will be 'myFT Digest'
+
+If a `title` property is provided in the event, the email subject will be `title` prefixed with the string 'myFT'. E.g. if `title` is 'Daily digest' the subject will be 'myFT Daily Digest'.
+
+If a `subject` property is provided in the event, this will be used the email subject. `subject` trumps `title`.
```
Comment (keirog) [inline README.md]: `used the email`->`used in the email`?

File: functions/create-email/lib/email/get-instant-subject.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+'use strict';
+module.exports = data => `myFT Instant Alert — ${data.concept.name} — ${data.headline}`;
```
Comment (keirog) [inline functions/create-email/lib/email/get-instant-subject.js]: I thought we were going for a colon after `Alert`?

**Comment by keirog** [comment]
👍 ☀️

**Pull request merged**